data.json

```diff
@@ -330,5 +330,15 @@
     "github": "https://github.com/Jisan-mia/dom-projects/tree/main/projects/kanban/",
     "difficulty": "Beginner",
     "tags": ["html", "css", "js"]
+  },
+  {
+    "id": 33,
+    "name": "Car Game",
+    "description": "A simple Car Game.",
+    "image": "https://github.com/user-attachments/assets/75294c6e-34cd-4d2d-b8e8-c77f2b15c732",
```

Comment:
this url gives me 404 error...
can you update the pr with a valid image URL or update it with below link:
https://github-production-user-asset-6210df.s3.amazonaws.com/61211600/435483715-783efad6-f231-46fc-a4ba-13bb7b8779f3.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20250420%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250420T165011Z&X-Amz-Expires=300&X-Amz-Signature=b2d86516359ede297947330b33ae7b5a3fc58224f4e60000145303cacd16c92e&X-Amz-SignedHeaders=host